tests/unit/oidc/test_services.py

```diff
@@ -937,22 +963,20 @@ class TestPyJWTBackstop:
     "Backstop" tests against unexpected PyJWT API changes.
     """
 
-    def test_decodes_token_bare_key(self):
-        privkey = rsa.generate_private_key(65537, 2048)
-        pubkey = privkey.public_key()
+    def __init__(self) -> None:
+        self._privkey = rsa.generate_private_key(65537, 2048)
+        self._pubkey = self._privkey.public_key()
```

Comment:
Diff is messy here, but I took the opportunity to optimize this test group a bit (RSA keygen is pretty slow and the keys don't need to be unique per-test, so I'm having them reuse the same keypair rather than regenerating each time.)